src/app.js

```diff
@@ -65,6 +65,11 @@ function listen (koaApp, config, ledgers) {
       log('app').error(error.message)
       process.exit(1)
     }
+    const ledgerUirs = Object.keys(ledgers.getLedgers())
+    for (let i = 0; i < ledgerUirs.length; i++) {
```

Comment:
Yeah, I also prefer a more functional approach, but I was having trouble getting the generators to work with the callback for `map`. I could do something like this:
```
    for (let ledgerUri of Object.keys(ledgers.getLedgers())) {
      const ledger = ledgers.getLedger(ledgerUri)
      yield ledger.connect()
    }
```
but I'm open to suggestions.